src/Assets/Bugsnag/Editor/BugsnagEditor.cs

```diff
@@ -224,5 +253,125 @@ public static void OnPostProcessBuild(BuildTarget target, string path)
             File.WriteAllLines(pbxPath, lines.ToArray());
         }
 #endif
+
+        public static void EnableEDM()
+        {
+            try
+            {
+                var path = Application.dataPath + EDMDepsFilePath;
+
+                File.WriteAllText(path, ANDROID_DEPS_XML);
+
+                foreach (var lib in GetKotlinLibs())
+                {
+                    lib.SetCompatibleWithPlatform(BuildTarget.Android, false);
+                    lib.SaveAndReimport();
+                }
+
+                AssetDatabase.Refresh();
```

Comment:
Suggest abstracting this to a `UpdateLibraryImportSettings(bool)` private method with it's own try-catch (if required) that can be shared with `DisableEDM` and make this function do fewer things.

Reply:
Done 👍